index.js

```diff
@@ -14,14 +22,24 @@ function sortByStringLength() {}
  * Example: "Hello"
  * [ 'elloH', 'lloHe', 'loHel', 'oHell', 'Hello' ]
  */
-function textScroller() {}
+function textScroller(word) {
+  let newArr = []
+  let length = word.length
+  for (let letter = 0; letter < length; letter++){
+    newArr.unshift(word.slice(length-letter,length) + word.slice(0,length-letter))
```

Comment:
Here again I like what you're trying to do but to be honest I think it makes this even harder to read.  `length` and `word.length` are so similar for me I wouldn't create a placeholder variable.

Additionally - I'd recommend _almost always instantiating for loops with i_.  It is such an established convention that you're almost making it more difficult to read by not including it.